code/modules/mod/mod_actions.dm

```diff
@@ -2,29 +2,28 @@
 	background_icon_state = "bg_tech_blue"
 	icon_icon = 'icons/mob/actions/actions_mod.dmi'
 	check_flags = AB_CHECK_CONSCIOUS
+	var/ai_action = FALSE
 	var/obj/item/mod/control/mod
 
 /datum/action/item_action/mod/New(Target)
 	..()
 	mod = Target
+	if(ai_action)
+		background_icon_state = ACTION_BUTTON_DEFAULT_BACKGROUND
 
 /datum/action/item_action/mod/Grant(mob/M)
```

Comment:
I hate being unable to suggest changes on lines that have "been deleted", screw you github.
```suggestion
/datum/action/item_action/mod/Grant(mob/action_receiver)
```